sdk/test-utils/recorder-new/src/sanitizer.ts

```
@@ -74,12 +75,13 @@ export class Sanitizer {
         const replacers = options[prop];
         if (replacers) {
           return Promise.all(
-            replacers.map((replacer: unknown) =>
-              this.addSanitizer({
+            replacers.map((replacer: RegexSanitizer) => {
+              if (!replacer.regex) return;
```

should we throw if it's undefined in live/record mode?

---

hmmm, good idea

---

only record mode